lib/queuePopulator/LogReader.js

```diff
@@ -444,6 +444,13 @@ class LogReader {
         if (!record || !record.entries) {
             return done();
         }
+        // Method 10 is APPEND_TO_LOG in Metadata.
+        // For now, operations with this method are used for communication between
+        // metadata-migration and SUR, and between SUR ingestances.
```

Comment:
"ingestance"? :slightly_smiling_face:

Reply:
I was probably trying to write "ingest instances" :laughing: